packages/orchestrator-ui-components/src/components/WFOSubscription/SubscriptionActions.tsx

```diff
@@ -36,7 +36,7 @@ export type SubscriptionActionsProps = {
     subscriptionId: string;
 };
 
-export const WFOSubscriptionActions: FC<SubscriptionActionsProps> = ({
+export const SubscriptionActions: FC<SubscriptionActionsProps> = ({
```

Comment:
Most of the other (sub)components have a WFO prefix; so why is it removed here?

Reply:
The idea is that the WFO prefix is for components that are - or can at some point can be - exposed to the  consumer of the packages and these will not be. Only the complete WFOsubscriptionDeatail component

Reply:
I do agree with keeping the WFO prefix since we might want to expose all components and its sub components. A consumer might decide to build his own WFOSubscription with, for example this component.